WORKSPACE

```diff
@@ -8,10 +8,6 @@ load("//webpack:dependencies.bzl", "rules_webpack_dependencies")
 
 rules_webpack_dependencies()
 
-load("@aspect_rules_js//js:repositories.bzl", "rules_js_dependencies")
-
-rules_js_dependencies()
```

Comment:
Why were all these removed?

Reply:
They are not needed since transitive closure of deps are all included in `rules_webpack_dependencies()` for rules_js downstream rules.

Reply:
Ahh. Removing it found a missing transitive dep on `bazel_features` in rules_js 1.34.0 that is add to rules_webpack_dependencies  along with this change.

https://github.com/aspect-build/rules_js/blob/v1.34.0/MODULE.bazel#L12

Reply:
[rules_webpack_dependencies](https://github.com/aspect-build/rules_webpack/blob/a83b2ff33ea7402a05f5d31b65ff84c4894488b0/webpack/dependencies.bzl#L10-L42) doesn't invoke `rules_js_dependencies()` though?

Reply:
That right. It can't because the load statement wouldn't work in that .bzl file since that is where the rules_js http_archive is. The pattern we've used in the derivate rule sets is that for the minimum version of rules_js they require the transitive WORKSPACE deps, such as rules_nodejs, are listed explicitly in the rules_foo_dependencies so that users don't have to call both rules_foo_dependencies and rules_js_dependencies in the WORKSPACE file. Tho this is all moot at this point since bzlmod solves the transitive deps story.